contracts/BaseRelayRecipient.sol

```diff
@@ -23,8 +23,8 @@ contract BaseRelayRecipient is IRelayRecipient {
         _;
     }
 
-    function getTrustedForwarder() public override view returns(address) {
-        return trustedForwarder;
+    function isTrustedForwarder(address forwarder) public override view returns(bool) {
```

Comment:
do you insist on bundling  "remove canRelay" with "remove getTrustedForwarder" ? I thnik they are separate issues.

Reply:
I don't, but the 'canRelay' used to call 'get forwarder' so it seemed to be related